linode/manager
# change: [M3-8817] - Replace PageSize autocomplete with simple select

## Description 📝
One less autocomplete! 

Main issue with this one was that it allowed the user to search options which did not make a whole lotta sense.

Using a simple select Textfield makes the most sense here, and does not affect the experience.

⚠️ using `change` for the changeset instead of a tech story since the user experience is slightly modified (not able to search the select anymore)

## Changes  🔄
- Replace PageSize autocomplete with simple select

## Preview 📷
![Screenshot 2024-11-14 at 17 35 02](https://github.com/user-attachments/assets/8991b658-505d-4f2c-b65e-8d68082c885f)


## How to test 🧪

### Verification steps
- Confirm the PaginationFooter pageSize select behaves with parity and without styling regression
- Update related e2e tests

## As an Author I have considered 🤔

*Check all that apply*
